CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Added
```

Comment:
"Added" is not the correct section. Should be "changed" or "fixed", but since this is also a breaking change, marking it as such would be appreciated.